powered_by.md

```diff
@@ -154,6 +154,7 @@ short description of your use case.
 * **[pandas][12]:** data analysis toolkit for Python programmers. pandas
   supports reading and writing Parquet files using pyarrow. Several pandas
   core developers are also contributors to Apache Arrow.
+* **[Parseable][51]:** Log analytics platform built for scale and usability. Ingest logs from anywhere and unify logs with Parseable. 
```

Comment:
How to use Apache Arrow?

Reply:
It uses arrow Recordbatch and flight as intermediary data format. https://github.com/parseablehq/parseable/blob/main/server/src/utils/arrow.rs

Reply:
Thanks. Could you add it too?

Reply:
done!